### PR TITLE
Add some namespace management using namespace-config-operator

### PR DESCRIPTION
**Pull request by etsauer**
@jkupferer

### CODE REVIEW

File: templates/cluster-gitops.yaml [hunk 1]
```diff
@@ -45,10 +45,10 @@ parameters:
   - name: CLUSTER_CONFIG_NAMESPACE
     value: cluster-config
   - name: CLUSTER_CONFIG_REPO_URL
-    value: https://github.com/redhat-cop/operationalizing-openshift-lab
+    value: https://github.com//operationalizing-openshift-lab
```
Comment (jkupferer) [inline templates/cluster-gitops.yaml]: What happened here?

Comment (etsauer) [reply templates/cluster-gitops.yaml]: no idea... i put it back.

File: manifests/namespace-config-operator-ns.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (jkupferer) [inline manifests/namespace-config-operator-ns.yaml]: Missing `---`

File: manifests/project-config/default-project-template.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
+apiVersion: template.openshift.io/v1
```
Comment (jkupferer) [inline manifests/project-config/default-project-template.yaml]: Missing `---`

File: manifests/project-config/multitenant-networkpolicy.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
```
Comment (jkupferer) [inline manifests/project-config/multitenant-networkpolicy.yaml]: Missing `---`

File: manifests/project-config/multitenant-networkpolicy.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      podSelector: {}
+      ingress:
+      - from:
+        - podSelector: {}
+  - apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy
+    metadata:
+      name: allow-network-policy-group-ingress
+    spec:
+      podSelector:
```
Comment (jkupferer) [inline manifests/project-config/multitenant-networkpolicy.yaml]: Can we do `podSelector: {}` here too?

**Pull request merged**